lang/go/go.talon

```diff
@@ -128,11 +201,11 @@ interface <user.text> [over]:
     insert(user.formatted_text(text, "PUBLIC_CAMEL_CASE"))
 
 state string: " string "
-[state] (int | integer | ant): "int"
+[state] (int | integer): "int"
 state slice: " []"
 slice of: "[]"
-[state] (no | nil): "nil"
-state (int | integer | ant) 64: " int64 "
+[state] (nil): "nil"
+state (int | integer) 64: " int64 "
```

Comment:
I find the "no" helps accuracy for me.  Presumably the "ant" is there for the same reason